### ISSUE TITLE
Error Animation "Scroll button for website"

### ISSUE THREAD

**Issue by tesye30** [tesye30/My-web-site#1]
<!DOCTYPE html>
 <html lang="en">
 <head>
    <meta charset="UTF-8">
     <title>App landig page de design</title>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
     
     <style type="text/css">
         
         .portal-home {
            background-image: -webkit-linear-gradient(145deg,#2DC6E5 0%, #1C2B2E 100%);
            position: relative;
            padding-top: 185px;
            padding-bottom: 180px;
            overflow: hidden;
            z-index: 1;
         }
         .col-header-text {
             color: white;    
         }  
         .col-header-text h1{
             font-size: 60px;
             line-height: 70px;
             letter-spacing: 0.05;
             font-weight: 300;
             color: white;   
         }
         .col-header-text h1 span{
             font-weight: 600;
         }
         .col-header-text p{
             font-size: 18px;
             line-height: 30px;
             margin: 25px 0 54px;
             text-align: justify;   
         }
         .col-header-img{
             position: relative;           
         }
         .col-header-img img{
             position: absolute;
         } 
         .col-header-img img.img-header-lg{
             right: -228px;
             top: -9rem;
             z-index: 20;
             
         } 
          .col-header-img img.img-header-sm{
             right: -304px;
             top: 6rem;
             z-index: 20;   
         } 
         .banner_btn {
             font-size: 14px;
             line-height: 58px;
             text-align: center !important;
             font-family: tahoma;
             font-weight: 700;
             width: 220px;
             height: 60px;
             display: inline-block;
             text-transform: uppercase;
             border-radius: 50px;
             border: 2px solid transparent;
             transition: all 0.5s linear;
             color: #272929;
             background: white;
             margin-right: 10px;
         } 
         .banner_btn:hover{
             text-decoration: none;
         }
         .social {
             margin-top: 90px;
         }  
         .social a{
             color: white;
             font-size: 20px;
             margin-right: 10px;
             margin-left: 20px;
         }
         
         .navbar{
             padding: 24px 0;
             border-bottom: none !important;
             transition: all 0.5s ease-in-out;
             
         }
         
         .navbar ul li a{
             color: white !important;
             text-transform: uppercase;
             font-weight: 600;
             letter-spacing: 1px;
             transition: all 0.5s ease-in-out;
             background: transparent !important;
         }
         
         .navbar ul li a:hover, .navbar ul li a:focus{
             color: deepskyblue !important;
             background: none !important;  
             
         }
         
         .navbar-toggle{
             border: 1px solid white !important;
             color: white !important;
             transition: all 0.5s ease-in;  
         }
         
         .icon-bar{
             color: white !important;
             background-color: white !important;
         }
         
         .navbar-toggle:hover{
             background-color: deepskyblue !important;
             
         }
         
         .navbar-inverse .navbar-collapse{
             background-color: transparent !important;
             
             
         }
         
         .location{
             border: 1px solid white;
             border-radius: 50px;
         }
         
         .navbar-inverse{
             background-color: transparent !important;
             
         }
         .navbar-brand{
             color: white;
                 font-weight: bold;
         }
         .navbar-brand:hover{
             color: white;
         }
         
         p{
             margin-left: -55px;
         }
         
     @keyframes animouse{
             
             0%{
                 opacity: 1;
                 top: 29%;   
             }
             15%{
                 opacity: 1;
                 top: 50%;   
             }
             50%{
                 opacity: 0;
                 top: 50%;   
             }
             100%{
                 opacity: 0;
                 top: 50%;   
             }
             
         }
         
         .scroll-btn{
             display: block;
             left: 0;
             right: 0;
             text-align: center;
             margin-top: -30px;
             
         }
         
         .scroll-btn > *{
             display: inline-block;
             line-height: 18px;
             font-size: 13px;
             font-weight: normal;
             color: #7c8c8d;
             color: #ffffff;
             font-family: tahoma;
             letter-spacing: 2px;
         }
         
         .scroll-btn > *:hover,
         .scroll-btn > *:focus,
         .scroll-btn > *:active,
         .scroll-btn > *.active{
                opacity: 0.8;
                filter: alpha(opacity=80)
             
         }
         
         .scroll-btn .mouse{
             
             position: relative;
             display: block;
             width: 35px;
             height: 55px;
             margin: 0 auto 20px;
             box-sizing: border-box;
             border: 3px solid white;
             border-radius: 23px;
         }
         
         scroll-btn .mouse > *{
             
             position: absolute;
             display: block;
             top: 29%;
             left: 50%;
             width: 8px;
             height: 8px;
             margin: -4px 0 0 -4px;
             background: white;
             border-radius: 50%;
             animation: animouse 2.5s linear infinite;
         }
         
    
         
     </style>
     
     </head>
     <body>
         
         <nav class="navbar navbar-inverse navbar-fixed-top">
            <div class="container">
                <div class="navbar-header">
                    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navi">
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                    
                    </button>
                    <a href="" class="navbar-brand">APP-KART</a>
                    
                    
                </div>
                <div class="collapse navbar-collapse" id="navi">
                <ul class="nav navbar-nav navbar-right">
                
                    <li><a href="">Home</a></li>
                    <li><a href="">Our Services</a></li>
                    <li><a href="">Works</a></li>
                    <li><a href="">Team</a></li>
                    <li><a href="">About us</a></li>
                    <li><a href="">Blog</a></li>
                    <li><a href="">Carrier</a></li>
                    <li><a href="" class="location">Our Location</a></li>
                </ul>
             </div>
                
             </div>
        
         </nav>
         
         <section id="home" class="portal-home">
         
         <div class="container">
            <div class="row">
                <div class="col-sm-8 col-header-text lr-padding">
                    <h1> O <span>portal</span> do site</h1>
                       <p> A Lipsum é um texto usado como um substituto na ausência do texto final de um modelo ou um site. Lipsum vem da contração dos textos alternativos mais conhecidos: "" Lorem ipsum "". Este tipo de texto também é chamado de texto em branco, texto falso, falso, preencher texto de substituição de texto.
                           
                           </p>
                            <a href="" class="banner_btn btn-white">Adiquira agora o app</a>
                            <a href="" class="banner_btn btn-transparent">Adiquira agora o app</a> 
                    </div>
                    <div class="col-sm-5 col-md-offset-0 col-md-3 col-header-img right-padding">
                    
                        <img src="home-pc.png" class="img-header-lg" alt="">
                        <img src="web.png" class="img-header-sm" alt="">

                    </div>
                </div>
             <div class="social">
                <a href=""><i class="fa fa-facebook"></i></a>
                <a href=""><i class="fa fa-twitter"></i></a>
                <a href=""><i class="fa fa-instagram"></i></a>
             </div>
         </div>
            <span class="scroll-btn">
                <a href="">
                     <span class="mouse">
                            <span></span>
                    </span>   
                </a>
             <p>Scroll me</p>
             </span>
              
     </section>
         
 </body>
 </html>
